assets/javascripts/admintable.js

```diff
@@ -166,7 +174,10 @@ function getAdminTableRowData(trElement, dataToSubmit, internalRowData) {
                     var key = this.substr(0, equationSignIndex);
                     var val = this.substr(equationSignIndex + 1);
                     settingsToSubmit[key] = val;
-                    internalRowSettings.push({key: key, value: val});
```

Comment:
Seems like it really doesn't like writing objects in a compact way. Maybe there's a rule/threshold to change that?

Reply:
that would be https://github.com/os-autoinst/openQA/pull/3268#discussion_r456940506 with the only additional of additional spaces if that's ok